src/Catalog/Helpers/CatalogWriterHelper.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NuGet.Services.Metadata.Catalog.Persistence;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    /// <summary>
+    /// Helper methods for writing to the catalog.
+    /// </summary>
+    public static class CatalogWriterHelper
+    {
+        /// <summary>
+        /// Asynchronously writes package metadata to the catalog.
+        /// </summary>
+        /// <param name="packageCatalogItemCreator">A package catalog item creator.</param>
+        /// <param name="packages">Packages to download metadata for.</param>
+        /// <param name="storage">Storage.</param>
+        /// <param name="lastCreated">The catalog's last created datetime.</param>
+        /// <param name="lastEdited">The catalog's last edited datetime.</param>
+        /// <param name="lastDeleted">The catalog's last deleted datetime.</param>
+        /// <param name="maxDegreeOfParallelism">The maximum degree of parallelism for package processing.</param>
+        /// <param name="createdPackages"><c>true</c> to include created packages; otherwise, <c>false</c>.</param>
```

Comment:
Why not called includeCreatedPackages instead of createdPackages?

Reply:
This is existing code which just moved to another type.